dolphinscheduler-service/src/main/java/org/apache/dolphinscheduler/service/process/ProcessService.java

```diff
@@ -1809,42 +1889,46 @@ public String getTaskWorkerGroup(TaskInstance taskInstance) {
 
     /**
      * get user by user id
+     *
      * @param userId user id
      * @return User
      */
-    public User getUserById(int userId){
+    public User getUserById(int userId) {
         return userMapper.selectById(userId);
     }
 
     /**
      * get resource by resoruce id
+     *
      * @param resoruceId resource id
      * @return Resource
      */
-    public Resource getResourceById(int resoruceId){
+    public Resource getResourceById(int resoruceId) {
         return resourceMapper.selectById(resoruceId);
     }
 
 
     /**
      * list resources by ids
+     *
      * @param resIds resIds
      * @return resource list
      */
-    public List<Resource> listResourceByIds(Integer[] resIds){
+    public List<Resource> listResourceByIds(Integer[] resIds) {
         return resourceMapper.listResourceByIds(resIds);
     }
 
     /**
      * format task app id in task instance
+     *
      * @param taskInstance
      * @return
      */
-    public String formatTaskAppId(TaskInstance taskInstance){
+    public String formatTaskAppId(TaskInstance taskInstance) {
         ProcessDefinition definition = this.findProcessDefineById(taskInstance.getProcessDefinitionId());
         ProcessInstance processInstanceById = this.findProcessInstanceById(taskInstance.getProcessInstanceId());
 
-        if(definition == null || processInstanceById == null){
+        if (definition == null || processInstanceById == null) {
```

Comment:
Hi,

Please don't change so many code format problems unrelated to your Pull Request. If you want to format the code, you can commit a new Pull Request.

Reply:
ok I'll fix it in next commit